### PR TITLE
pan-os Documentation and metadata improvements

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] In Progress

## Related Issues
fixes: https://github.com/demisto/etc/issues/30682

## Minimum version of Demisto
- [x] 3.5.0

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/PAN-OS/Integrations/Panorama/Panorama.yml [hunk 1]
```diff
@@ -30,16 +30,19 @@ configuration:
   name: template
   required: false
   type: 0
-- display: Use URL Filtering for auto enrichment
+- additionalinfo: "add here"
```
Comment (amshamah419) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.yml:33]: :ma:

File: Packs/PAN-OS/Integrations/Panorama/README.md [hunk 2]
```diff
@@ -14,6 +14,7 @@ This integration was integrated and tested with version 8.1.0 and 9.0.1 of Palo
 ## Use Cases
 * Create custom security rules in Palo Alto Networks PAN-OS.
 * Creating and updating address objects, address-groups, custom URL categories, URL filtering objects.
+* Use the URL Filtering category information from Palo Alto to enrich URLs by checking the *use_url_filtering* parameter. A proper licnese on the Firewall is needed.
```
Comment (amshamah419) [inline Packs/PAN-OS/Integrations/Panorama/README.md:17]:
```suggestion
* Use the URL Filtering category information from Palo Alto Networks to enrich URLs by checking the *use_url_filtering* parameter. A valid license for the Firewall is required.
```

File: Packs/PAN-OS/Integrations/Panorama/Panorama.yml (hunk 1 above)
Comment (bakatzir) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.yml:33]: @kirbles19 will review, more data in the slack thread referenced in the issue

Comment (amshamah419) [reply Packs/PAN-OS/Integrations/Panorama/Panorama.yml:33]: Okay cool

Comment (kirbles19) [inline Packs/PAN-OS/Integrations/Panorama/Panorama.yml]: If selected, when running the !url command, the command will execute using pan-os with PAN_DB (with applied filters). The URL filtering categories determine DBot score (malicious, suspicious, benign).

**Review by kirbles19 — APPROVED** [review]

**Pull request merged**